variables.tf

```diff
@@ -74,10 +74,11 @@ variable "enable_private_default_route" {
 
 variable "flow_logs" {
   type = object({
```

Comment:
OK, so all the layout seems to be in order now. Only thing I notice now (looking at the logic itself) is that you choose a name that seems a bit off. I think it's clearer to use `iam_role_permission_boundary` as field name so its more clear what it actually matched to. Same as with `iam_role_name` which is the name or the IAM role where this is the permission boundary of the IAM role.

Reply:
renamed.